pgxn/neon/communicator/src/neon_request.rs

```diff
@@ -174,6 +181,27 @@ pub struct CGetPageVRequest {
     pub dest: [ShmemBuf; MAX_GETPAGEV_PAGES],
 }
 
+#[repr(C)]
+#[derive(Copy, Clone, Debug)]
+pub struct CReadSlruSegmentRequest {
+    pub request_id: u64,
+    pub slru_kind: SlruKind,
+    pub segment_number: u32,
+    pub request_lsn: CLsn,
+    /// Must be a null-terminated C string containing **absolute**
+    /// file path.
+    pub destination_file_path: ShmemBuf,
+}
+
+impl CReadSlruSegmentRequest {
+    /// Returns the path to the file where the segment is read into.
```

Comment:
Wording feels a little awkward to me. Maybe:
```suggestion
    /// Returns file path where the communicator will write the SLRU segment.
```